packages/react/src/Logo/brand/LogoMuseumWeesp.tsx

```diff
@@ -0,0 +1,45 @@
+import { forwardRef } from 'react'
+import type { ForwardedRef, SVGProps } from 'react'
+
+const LogoMuseumWeesp = forwardRef((props: SVGProps<SVGSVGElement>, ref: ForwardedRef<SVGSVGElement>) => (
```

Comment:
Don’t we generate the logo components from the SVG sources yet, like we do with the icons?

The plain SVG is missing from `proprietary/assets/logo`.

Reply:
We're talking about the same ticket right? DES-589?

Reply:
It wasn’t, just a question, I could have looked up the answer myself. So please add the SVG for consistency and don’t worry about generating components.